changes/v4.3.22-en.md

```diff
@@ -5,16 +5,18 @@
 
 ### Bug fixes
 
-- Improve the display of rule's 'Maximum Speed' counter to only reserve 2 decimal places. [#9185](https://github.com/emqx/emqx/pull/9185)
+- Fix that after uploading a backup file with an UTF8 filename, HTTP API `GET /data/export` fails with status code 500 [#9224](https://github.com/emqx/emqx/pull/9224).
+
+- Improve the display of rule's 'Maximum Speed' counter to only reserve 2 decimal places [#9185](https://github.com/emqx/emqx/pull/9185).
   This is to avoid displaying floats like `0.30000000000000004` on the dashboard.
```

Comment:
```suggestion
- Improve the display of rule's 'Maximum Speed' counter [#9185](https://github.com/emqx/emqx/pull/9185).
  Keep only two digits to the right of the decimal point, to avoid displaying floats like `0.30000000000000004` on the dashboard.
```